### DISCUSSION TITLE
Upgrade from NB-store 3.1.3

### DISCUSSION THREAD

**Discussion by Nicbobbo6** [openstore-ecommerce/OpenStore#220] [Q&A] [answered]
Hello,
I'm currently upgrading a DNN instance from 7.3 to latest, and i want to upgrade my store too. I checked the txt doc, but to do the upgrade, i need to backup my actual store, and i can't find the way to do that.
Maybe someone can help me ?
Thank you

**Comment by DNNMonster** [comment]
I believe this means you should export all your store data to flat files using the Back Office admin export tool. You can then use it's partner import tool to populate the v4 site when your upgrade is complete.   The term backup may be a little confusing here.  I've been on v4 for some time so others will speak up if I'm off target.  You're gonna like v4.

**Reply by Nicbobbo6** [reply]
Thank you for you're response. I can't find the export tool in the admin part of the menu. Can you tell me where it should be in this menu ?
Thanks !
![image](https://user-images.githubusercontent.com/25977189/146514577-2931c820-578c-463e-85ce-7b9138e4f955.png)

**Comment by DNNMonster** [comment]
I think we need another v3 user to jump in here....   I assumed the export/import features were available but I could be wrong.

**Reply by Nicbobbo6** [reply]
Thanks again for you're time, i will try to upgrade to the latest V3, but i need to make the package because it is not available as release on GitHub. Maybe this function was added after v3.1.3...

**Comment by in8dna** [comment]
This is the latest version I have. maybe it'll help.
[NBrightStore_03.06.06.03_DNN8_Install.zip](https://github.com/openstore-ecommerce/OpenStore/files/7774928/NBrightStore_03.06.06.03_DNN8_Install.zip)

**Comment by stetard** [comment, marked as answer]
Thanks for this package.
After studying the OpenStore documentation, we understood that the back office wasn't accessible through the module itself but by a page using the dedicated skin provided.
Now we have exported data and installed OpenStore in its latest version. But OpenStore seems to retrieve all data without needing to import them. Is it correct for you ?
Best regards
